android/src/main/java/com/facebook/react/views/modal/RNGHModalUtils.java

```diff
@@ -0,0 +1,21 @@
+package com.facebook.react.views.modal;
+
+import android.view.MotionEvent;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+
+/**
+ * For handling gestures inside RNGH we need to have access to some methods of
+ * `ReactModalHostView.DialogRootViewGroup`. This class is not available outside
+ * package so this file exports important features.
+ */
+
+public class RNGHModalUtils {
+  static public void dialogRootViewGroupOnChildStartedNativeGesture(ViewGroup modal, MotionEvent androidEvent) {
```

Comment:
Could you please change `static public` to `public static`? It's more common to see the latter in Java.